repoman/setup.py

```diff
@@ -393,6 +398,17 @@ def run(self):
 		print('Symlinking %s -> %s' % (conf_dir, conf_src))
 		os.symlink(conf_src, conf_dir)
 
+		# symlink 'pym' directory
+		pym_dir = os.path.join(self.top_dir, 'pym')
+		if os.path.exists(pym_dir):
+			if not os.path.islink(pym_dir):
+				raise SystemError('%s exists and is not a symlink (collision)'
+					% repr(pym_dir))
+			os.unlink(pym_dir)
+		pym_src = 'lib'
```

Comment:
But why `lib`? Can't we either fix the stuff that requires it to use `pym`, or finally drop the senseless `pym` if we need to have `lib` anyway?

Reply:
Yeah, it doesn't matter what the name of the directory is, because the code doing the sys.path insert has direct access to the actual directory name and therefor it doesn't need to make an assumption.

Reply:
I forgot that the files in the executable are not nested in the module structure, so they really don't have a way of knowing pym vs. lib. I'd really like to eliminate the sys.path inserts and just use PYTHONPATH when when tweaks are needed.

Reply:
In fact, PYTHONPATH is perfectly reliable for the purposes of unit tests, since the unit test code can easily export PYTHONPATH for use by any executables called in subprocesses.